crates/bevy_ecs/src/change_detection.rs

```diff
@@ -542,6 +542,45 @@ impl<'a, T: ?Sized> Ref<'a, T> {
     pub fn into_inner(self) -> &'a T {
         self.value
     }
+    /// Map `Ref` to a different type using `f`.
+    ///
+    /// Note that unlike [`Map::map_unchanged`] this doesn't do anything
+    /// special. It's just a more convinient way of constructing a new `Ref`
+    /// from an existing one.
```

Comment:
Nit: Not sure this is super necessary since users probably know they can just use `new` instead (which I’m guessing is what makes this "do nothing special"). Might be worth noting,though, that this won't trigger or reset the change detection state.